docusaurus.config.js

```diff
@@ -239,6 +275,48 @@ module.exports = {
       appId: "6KYF3VMCXZ",
       indexName: "camunda",
     },
+    languageTabs: [
+      {
+        highlight: "bash",
+        language: "curl",
+        logoClass: "bash",
+      },
+      {
+        highlight: "java",
+        language: "java",
+        logoClass: "java",
+        variant: "okhttp",
+        variants: ["okhttp", "unirest"],
+      },
+      {
+        highlight: "javascript",
+        language: "nodejs",
+        logoClass: "nodejs",
+        variant: "native",
+        variants: ["native", "axios", "request", "unirest"],
+      },
+      {
+        highlight: "go",
+        language: "go",
+        logoClass: "go",
+        variant: "native",
+        variants: ["native", "", " ", "  "],
+      },
+      {
+        highlight: "python",
+        language: "python",
+        logoClass: "python",
+        variant: "requests",
+        variants: ["requests", "http.client"],
+      },
+      {
+        highlight: "csharp",
+        language: "csharp",
+        logoClass: "csharp",
+        variant: "RestSharp",
+        variants: ["restsharp", "httpclient"],
+      },
+    ],
```

Comment:
@akeller here are the definitions of languages for sample code. I'm happy to change any of this if you have requests! I attempted to sort them based on what I understood to be popularity in the original implementation of this.

Reply:
Can you swap csharp and go?

Reply:
resolved in latest commit